app/controllers/links_controller.rb

```diff
@@ -35,9 +35,6 @@ def update
   def destroy
     if @link.make_missing
```

Comment:
Since this calls `save!` underneath, we don't need the `if` anymore either.

Reply:
Ahh yes, good catch 👍